libecl/src/ecl_grid.c

```diff
@@ -1066,6 +1066,21 @@ static void ecl_cell_taint_cell( ecl_cell_type * cell ) {
 }
 
 
+static int ecl_cell_get_twist( const ecl_cell_type * cell ) {
+  double sign = 1.0;
+  int twist_count = 0;
+
+  for (int c = 0; c < 4; c++) {
+    const point_type * p1 = &cell->corner_list[c];
+    const point_type * p2 = &cell->corner_list[c + 4];
+    if (sign * (p2->z - p1->z) < 0)
+      twist_count += 1;
+  }
+  return twist_count;
+}
```

Comment:
yes, I understand that in principle, `sign` could be `-1` however, that's not going to happen unless we explicitly add some code somewhere somewhen.   Why not stick with the simpler version until we implement support for changing sign?